collections/_posts/2024-01-30-brain-imaging-generation.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: review
+title: "Brain Imaging Generation with Latent Diffusion Models"
+tags: diffusion model, generative model
+author: "Olivier Bernard, Celia Goujat"
+cite:
+    authors: "Walter H. L. Pinaya, Petru-Daniel Tudosiu, Jessica Dafflon, Pedro F Da Costa, Virginia Fernandez, Parashkev Nachev, Sebastien Ourselin, and M. Jorge Cardoso"
+    title: "Brain Imaging Generation with Latent Diffusion Models"
+    venue: "Arxiv 2022"
+pdf: "https://arxiv.org/pdf/2209.07162.pdf"
+---
+
+# Notes
+
+* The generated synthetic dataset is available at [Academics Torrents](https://academictorrents.com/details/63aeb864bbe2115ded0aa0d7d36334c026f0660b) 
+* An inspired code is available on the following [Monai link](https://monai.io/model-zoo.html)
+
+&nbsp;
+
+# Highlights
+
+* Application of the Latent Diffusion Model framework for MR image synthesis of the human brain
+* An encoder/decoder model dedicated to brain MRI reconstruction is proposed
+* Inverstigation of the application of conditioning to age, gender, ventricular volumes and brain volumes 
+
+* The model achieves new SOTA for brain MR image synthesis
+* A synthetic dataset of 100,000 volumes together with the conditioning information are publicly available
```

Comment:
I would say : A synthetic dataset of 100,000 volumes, along with the conditioning information, is publicly available

Reply:
Done